src/PJ_lcc.c

```diff
@@ -5,22 +5,22 @@
 #include "proj_math.h"
 
 PROJ_HEAD(lcc, "Lambert Conformal Conic")
-    "\n\tConic, Sph&Ell\n\tlat_1= and lat_2= or lat_0";
+    "\n\tConic, Sph&Ell\n\tlat_1= and lat_2= or lat_0, scale=";
```

Comment:
could you update  docs/source/operations/projections/lcc.rst to document scale ?

Reply:
Sure, see the next commit.